Dockerfile.ios

```
@@ -10,31 +10,46 @@ RUN dnf -y install --setopt=install_weak_deps=False \
+    usage_examples/ios_toolchain/build.sh /root/files/iPhoneSimulator${IOS_SDK}.sdk.tar.xz arm64 && \
+    mkdir -p /root/ioscross/arm64_sim && \
+    mv usage_examples/ios_toolchain/target/* /root/ioscross/arm64_sim && \
+    mkdir /root/ioscross/arm64_sim/usr && \
+    ln -s /root/ioscross/arm64_sim/bin /root/ioscross/arm64_sim/usr/bin && \
+    #iOS x86-64 simulator
     sed -i 's#^TRIPLE=.*#TRIPLE="x86_64-apple-darwin11"#' usage_examples/ios_toolchain/build.sh && \
     usage_examples/ios_toolchain/build.sh /root/files/iPhoneSimulator${IOS_SDK}.sdk.tar.xz x86_64 && \
-    mkdir -p /root/ioscross/x86_64 && \
-    mv usage_examples/ios_toolchain/target/* /root/ioscross/x86_64 && \
-    mkdir /root/ioscross/x86_64/usr && \
-    ln -s /root/ioscross/x86_64/bin /root/ioscross/x86_64/usr/bin
+    mkdir -p /root/ioscross/x86_64_sim && \
+    mv usage_examples/ios_toolchain/target/* /root/ioscross/x86_64_sim && \
+    mkdir /root/ioscross/x86_64_sim/usr && \
+    ln -s /root/ioscross/x86_64_sim/bin /root/ioscross/x86_64_sim/usr/bin
 
 ENV OSXCROSS_IOS=not_nothing
 ENV IOSCROSS_ROOT=/root/ioscross
-ENV PATH="/root/ioscross/arm64/bin:/root/ioscross/x86_64/bin:${PATH}"
+ENV PATH="/root/ioscross/arm64/bin:/root/ioscross/arm64_sim/bin:/root/ioscross/x86_64_sim/bin:${PATH}"
 
 RUN if [ -z "${mono_version}" ]; then echo -e "\n\nargument mono-version is mandatory!\n\n"; exit 1; fi && \
     cp -a /root/files/${mono_version} /root && \
     cd /root/${mono_version} && \
     cd godot-mono-builds && \
     export MONO_SOURCE_ROOT=/root/${mono_version} && \
-    python3 ios.py configure -j --verbose --target=arm64 --ios-toolchain ${IOSCROSS_ROOT}/arm64 --ios-sdk ${IOSCROSS_ROOT}/arm64/SDK/iPhoneOS${IOS_SDK}.sdk --osx-toolchain ${OSXCROSS_ROOT} && \
-    python3 ios.py make -j --verbose --target=arm64 && \
-    python3 ios.py configure -j --target=x86_64 --ios-toolchain ${IOSCROSS_ROOT}/x86_64 --ios-sdk ${IOSCROSS_ROOT}/x86_64/SDK/iPhoneOS${IOS_SDK}.sdk --osx-toolchain ${OSXCROSS_ROOT} && \
-    python3 ios.py make -j --target=x86_64 && \
+    #iOS arm64 device
+    python3 ios.py configure -j --verbose --is-sim=no --target=arm64 --ios-toolchain ${IOSCROSS_ROOT}/arm64 --ios-sdk ${IOSCROSS_ROOT}/arm64/SDK/iPhoneOS${IOS_SDK}.sdk --osx-toolchain ${OSXCROSS_ROOT} && \
+    python3 ios.py make -j --verbose --is-sim=no --target=arm64 && \
+    #iOS arm64 simulator
+    python3 ios.py configure -j --verbose --is-sim=yes --target=arm64 --ios-toolchain ${IOSCROSS_ROOT}/arm64_sim --ios-sdk ${IOSCROSS_ROOT}/arm64_sim/SDK/iPhoneOS${IOS_SDK}.sdk --osx-toolchain ${OSXCROSS_ROOT} && \
+    python3 ios.py make -j --verbose --is-sim=yes --target=arm64 && \
+    #iOS x86-64 simulator
+    python3 ios.py configure -j --verbose --is-sim=yes --target=x86_64 --ios-toolchain ${IOSCROSS_ROOT}/x86_64_sim --ios-sdk ${IOSCROSS_ROOT}/x86_64_sim/SDK/iPhoneOS${IOS_SDK}.sdk --osx-toolchain ${OSXCROSS_ROOT} && \
+    python3 ios.py make -j --verbose --is-sim=yes --target=x86_64 && \
```

Since godotengine/godot-mono-builds#39 superseded godotengine/godot-mono-builds#30, this changes are needed:

```suggestion
    #iOS arm64 device
    python3 ios.py configure -j --verbose --target=arm64 --ios-toolchain ${IOSCROSS_ROOT}/arm64 --ios-sdk ${IOSCROSS_ROOT}/arm64/SDK/iPhoneOS${IOS_SDK}.sdk --osx-toolchain ${OSXCROSS_ROOT} && \
    python3 ios.py make -j --verbose --target=arm64 && \
    #iOS arm64 simulator
    python3 ios.py configure -j --verbose --target=arm64-sim --ios-toolchain ${IOSCROSS_ROOT}/arm64_sim --ios-sdk ${IOSCROSS_ROOT}/arm64_sim/SDK/iPhoneOS${IOS_SDK}.sdk --osx-toolchain ${OSXCROSS_ROOT} && \
    python3 ios.py make -j --verbose --target=arm64-sim && \
    #iOS x86-64 simulator
    python3 ios.py configure -j --verbose --target=x86_64 --ios-toolchain ${IOSCROSS_ROOT}/x86_64_sim --ios-sdk ${IOSCROSS_ROOT}/x86_64_sim/SDK/iPhoneOS${IOS_SDK}.sdk --osx-toolchain ${OSXCROSS_ROOT} && \
    python3 ios.py make -j --verbose --target=x86_64 && \
```